etc/inc/disable-programs.inc

```diff
@@ -396,6 +398,8 @@ blacklist ${HOME}/.config/yandex-browser
 blacklist ${HOME}/.config/yandex-browser-beta
 blacklist ${HOME}/.config/yelp
 blacklist ${HOME}/.config/youtube-dl
+blacklist ${HOME}/.config/youtubemusic-nativefier-040164
```

Comment:
The "040164" look generated. Is it stable?

Reply:
I need to check I uninstalled the thing.

Reply:
Reinstalled still `youtubemusic-nativefier-040164`